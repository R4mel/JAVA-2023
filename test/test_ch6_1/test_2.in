4
1
s2 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
s4 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
2
w2 3 33.3 false :Kangnam-gu Seoul: Samsung Director
w4 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
0
0
