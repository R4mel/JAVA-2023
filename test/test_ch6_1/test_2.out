
******* Main Menu ********
* 0.exit 1.PersonManager *
* 2.ch2 3.ch3 4.ch5      *
**************************
menu item? 4

***** Inheritance Menu ******
* 0.exit 1.Student 2.Worker *
*****************************
menu item? 1
--------------------
p1: s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
p2: s2 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
p1.equals(p2) : false
--------------------
p1: s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
p2: s1 1 65.4 true :Jongno-gu Seoul: Physics 2.8 1
p1.equals(p2) : true
--------------------
p1: s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
p2: s1 1 65.4 true :Jongno-gu Seoul: Physics-Electronics 2.8 1
p1.equals(p2) : false
--------------------
p1: s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
p2: s1 1 65.4 true :Jongno-gu Seoul: Physics 2.8 2
p1.equals(p2) : false
--------------------
p1: s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
p2: s1 1 65.4 true :Jongno-gu Seoul: Physics 2.8 1
p1.equals(p2) : true
--------------------
~~~~~~~~~~~~~~~~ Student::whatAreYouDoing() ~~~~~~~~~~~~~~~~
s2 is studying as a 1-year student in Physics
s2 took several courses and got GPA 2.8
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

~~~~~~~~~~~~~~~~ Student::whatAreYouDoing() ~~~~~~~~~~~~~~~~
s2 is studying as a 1-year student in Physics
s2 took several courses and got GPA 2.8
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
s3: s2 1 65.4 true :Jongno-gu Seoul: Physics 2.8 1
--------------------
s2: s2 1 65.4 true :Jongno-gu Seoul: Physics 2.8 1
s2:  0 0.0 false ::  0.0 0
--------------------
input student: s2 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
s2: s2 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
--------------------
input new student: s4 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
s4: s4 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4

***** Inheritance Menu ******
* 0.exit 1.Student 2.Worker *
*****************************
menu item? 2
--------------------
p1: w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
p2: w2 3 33.3 false :Kangnam-gu Seoul: Samsung Director
p1.equals(p2) : false
--------------------
p1: w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
p2: w1 3 33.3 false :Kangnam-gu Seoul: Samsung-Hyundai Director
p1.equals(p2) : false
--------------------
p1: w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
p2: w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director-Manager
p1.equals(p2) : false
--------------------
p1: w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
p2: w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
p1.equals(p2) : true
--------------------
!!!!!!!!!!!!!!!! Worker::whatAreYouDoing()!!!!!!!!!!!!!!!!!
w2 works in Samsung as Director
w2 is now enjoying his(her) vacation.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

!!!!!!!!!!!!!!!! Worker::whatAreYouDoing()!!!!!!!!!!!!!!!!!
w2 works in Samsung as Director
w2 is now enjoying his(her) vacation.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
w3: w2 3 33.3 false :Kangnam-gu Seoul: Samsung Director
--------------------
w2: w2 3 33.3 false :Kangnam-gu Seoul: Samsung Director
w2:  0 0.0 false ::  
--------------------
input worker: w2 3 33.3 false :Kangnam-gu Seoul: Samsung Director
w2: w2 3 33.3 false :Kangnam-gu Seoul: Samsung Director
--------------------
input new worker: w4 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
w4: w4 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

***** Inheritance Menu ******
* 0.exit 1.Student 2.Worker *
*****************************
menu item? 0

******* Main Menu ********
* 0.exit 1.PersonManager *
* 2.ch2 3.ch3 4.ch5      *
**************************
menu item? 0

Good bye!!
